src/AcceptanceTests/When_using_databus_with_custom_provider.cs

```diff
@@ -43,7 +43,7 @@ public EndpointWithCustomProvider()
             {
                 EndpointSetup<DefaultServer>(config =>
                 {
-                    config.RegisterComponents(services => services.AddSingleton<IProvideBlobContainerClient, CustomProvider>());
+                    config.RegisterComponents(services => services.AddSingleton<IProvideBlobServiceClient, CustomProvider>());
```

Comment:
We still need to set the container here to not use the default

Reply:
which I won't do as I prefer the first approach and have it as part of a configuration class 💯